src/Benchmarks/Middleware/MultipleQueriesEfMiddleware.cs

```diff
@@ -21,20 +22,19 @@ public class MultipleQueriesEfMiddleware
         };
 
         private readonly RequestDelegate _next;
-        private readonly EfDb _db;
 
-        public MultipleQueriesEfMiddleware(RequestDelegate next, EfDb db)
+        public MultipleQueriesEfMiddleware(RequestDelegate next)
```

Comment:
When I run a benchmark with these changes, the first 50 or so requests work fine, but then RPS goes to 0.  If I revert this file, RPS is similar to the dev branch.